downstream/modules/eda/con-custom-credential-types.adoc

```diff
@@ -0,0 +1,61 @@
+[id="eda-custom-credential-types"]
+
+= Custom credential types
+
+As a system administrator, you can define a custom credential type in a standard format by using a YAML or JSON-like definition. You can define a custom credential type that works in ways similar to existing credential types. For example, a custom credential type can inject an API token for a third-party web service into an environment variable, for your rulebook or custom inventory script to consume.
+
+Each credential type displays its own unique configurations in the Input Configuration field and the Injector Configuration field, if applicable. Custom credentials support the following ways of injecting their authentication information:
+
+* Environment variables
+* Ansible extra variables
+* File-based templating, which means generating .ini or .conf files that contain credential values
+
+You can attach one SSH and multiple cloud credentials to a job template. Each cloud credential must be of a different type. Only one of each type of credential is permitted. Vault credentials and machine credentials are separate entities.
+
+[NOTE]
+====
+* When creating a new credential type, you must avoid collisions in the `extra_vars`, `env``, and file namespaces.
+* Environment variable or extra variable names must not start with *ANSIBLE_* because they are reserved.
```

Comment:
@jself-sudoku This should be *EDA* not ANSIBLE_

Reply:
Thanks, @mkanoor, for calling this one out. Will update.